service/models.py

```diff
@@ -15,26 +20,36 @@
 class DataValidationError(Exception):
     """ Used for an data validation errors when deserializing """
 
-    pass
-
 
-class YourResourceModel(db.Model):
+class Customer(db.Model):
     """
-    Class that represents a YourResourceModel
+    Class that represents a customers
     """
 
     app = None
 
     # Table Schema
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(63))
+    name = db.Column(db.String(63),nullable=False)
+    lastname = db.Column(db.String(63),nullable=False)
+    email = db.Column(db.String(120))
+    phone = db.Column(db.String(60))
+    address1 = db.Column(db.String(256))
```

Comment:
i think this should be street_line1 instead of address1 and address2 as it's ambiguous. A future developer might be expecting to receive the whole address when they ask for address1 / address2. Like in the ZH schema:
```
{
   "id": 
       {
   	"firstname": <First Name>,
   	"lastlame": <Last Name>,
   	"email": <Email Address>,
   	"tel": <Telephone Number>,
   	"street_line1": <Address Line 1>,
   	"street_line2": <Address Line 2>,
   	"city": <Address City>,
   	"state": <Address State>,
   	"country": <Address Country>,
   	"zip": <Address Zip Code>,
   	"created_at": <Date of creation>f
   	"last_updated": <Date of last update to record>	
       }
}
```